tests/testthat/test-settors.R

```diff
@@ -95,30 +95,37 @@ test_that("seconds settor returns NA for spring dst gap", {
 
 })
 
-test_that("seconds settor retains object class", {
+test_that("seconds settor retains object class for datetimes", {
   poslt <- as.POSIXlt("2010-02-14 01:59:59", tz = "GMT", format
      = "%Y-%m-%d %H:%M:%S")
   posct <- as.POSIXct(poslt)
-  date <- as.Date(poslt)
 
   second(poslt) <- 10
   second(posct) <- 10
-  day(date) <- 10
 
   expect_that(poslt, is_a("POSIXlt"))
   expect_that(posct, is_a("POSIXct"))
-  expect_that(date, is_a("Date"))
 
   second(poslt) <- 70
   second(posct) <- 70
-  day(date) <- 32
 
   expect_that(poslt, is_a("POSIXlt"))
   expect_that(posct, is_a("POSIXct"))
-  expect_that(date, is_a("Date"))
 
 })
 
+test_that("seconds settor makes POSTXct from Date", {
+  date <- as.Date("2010-02-14", tz = "GMT", format = "%Y-%m-%d")
+
+  second(date) <- 10
+
+  expect_that(date, is_a("POSIXct"))
```

Comment:
Can you please update to use modern style, i.e. `expect_s3_class(date, "POSIXct")`

Reply:
Sorry, I meant _just_ for the code that you touched. Do you mind undoing the changes above (to make it easier to remove), and only using `expect_s3_class()` in the tests that you have written?